clang/test/SemaSYCL/intel-fpga-local.cpp

```diff
@@ -298,10 +298,18 @@ void diagnostics()
   [[intel::merge("mrg4", "depths")]] unsigned int mrg_invalid_arg[4];
 
   // Checking of different argument values.
```

Comment:
Sorry I didn't mention this earlier but can you also add a test where arguments match. I know we have an AST test for this but it makes sense to endure diagnostics work as expected in this case too.

Reply:
Thanks @elizabethandrews for reviews and feedbacks. 

>>can you also add a test where arguments match.

Yes, i have added the test.